src/resources/ha-style.html

```diff
@@ -5,53 +5,37 @@
   body {
     font-size: 14px;
 
-    /* for paper-toggle-button */
-    --paper-grey-50: #fafafa;
-    --paper-grey-200: #eeeeee;
-
-    --paper-item-icon-color: #44739e;
-    --paper-item-icon-active-color: #FDD835;
-
+    --primary-color: #03A9F4; /* header */
     --dark-primary-color: #0288D1;
-    --primary-color: #03A9F4;
     --light-primary-color: #B3E5FC;
-    --text-primary-color: #ffffff;
-    --accent-color: #FF9800;
-    --primary-background-color: var(--paper-grey-50);
-    --secondary-background-color: #E5E5E5;
+    --text-primary-color: #FFFFFF; /* header */
+    --primary-background-color: #fafafa;
     --primary-text-color: #212121;
+
+    --secondary-background-color: #E5E5E5; /* states, */
     --secondary-text-color: #727272;
+
+    --tertiary-background-color: #FFFFFF; /* badges, state cards, side menu */
+    --tertiary-text-color: rgb(76, 76, 76); /* badges, */
+
     --disabled-text-color: #bdbdbd;
     --divider-color: rgba(0, 0, 0, .12);
+    --accent-color: #FF9800;
 
     --table-row-background-color: transparant;
-    --table-row-alternative-background-color: #eee;
+    --table-row-alternative-background-color: var(--secondary-background-color);
 
-    --paper-toggle-button-checked-ink-color: #039be5;
-    --paper-toggle-button-checked-button-color: #039be5;
-    --paper-toggle-button-checked-bar-color: #039be5;
-
-    --paper-slider-knob-color: var(--primary-color);
-    --paper-slider-knob-start-color: var(--primary-color);
-    --paper-slider-pin-color: var(--primary-color);
-    --paper-slider-active-color: var(--primary-color);
-    --paper-slider-secondary-color: var(--light-primary-color);
-    --paper-slider-container-color: var(--divider-color);
+    --paper-item-icon-color: #44739e;
+    --paper-item-icon-active-color: #FDD835;
 
-    --paper-card-background-color: #FFF;
-    --paper-listbox-background-color: #FFF;
+    --sidebar-text_-_background: var(--tertiary-background-color);
```

Comment:
It is a variable that was already used, I make --tertiary-background-color, so users can specify that and don't have to bother about the --sidebar-text_-_background. Do you want me to replace all the uses of --sidebar-text_-_background in the html with --tertiary-background-color?